lib/annotate/helpers.rb

```diff
@@ -15,10 +15,7 @@ def include_models?
       end
 
       def true?(val)
-        return false if val.blank?
-        return false unless val =~ Constants::TRUE_RE
-
-        true
+        val.present? && Constants::TRUE_RE.match?(val)
```

Comment:
```ruby
def true?(val)
  return false if val.blank?
  return false unless val =~ Constants::TRUE_RE
  true
```
can be refactored to
```ruby
def true?(val)
  if val.blank?
    false
  else
    if val =~ Constants::TRUE_RE
      true
    else
      false
    end
  end
end
```
So we can easily replace them to a one liner method.

I used `Regexp#match?` instead of `String#=~` because `String#=~` returns `nil` or an integer.

Reply:
👍 nice